.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
           shell: bash
           if: "${{ inputs.os == 'macos-latest' }}"
           run: |
-              brew install git gcc pkgconfig openssl redis
+              brew install git gcc pkgconfig openssl redis coreutils
```

Comment:
how is this relevant to the PR? please remove if not required.

Reply:
Same as here https://github.com/aws/glide-for-redis/pull/1044#discussion_r1504686462